GitSocial/GitSocial/GSButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  GSButton.swift
+//  GitSocial
+//
+//  Created by Dillon on 11/27/21.
+//
+
+import UIKit
+
+class GSButton: UIButton {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    /// Required init used with storyboards
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    /// Custom init for our button
+    init(backgroundColor: UIColor, title: String) {
+        super.init(frame: .zero)
+        self.backgroundColor = backgroundColor
+        self.setTitle(title, for: .normal)
+        configure()
+    }
+    
+    private func configure() {
+        layer.cornerRadius = 10
```

Comment:
Refer to next comment for context
```suggestion
        translatesAutoresizingMaskIntoConstraints = false
        layer.cornerRadius = 10
```

Reply:
Refer to issue #3